.appveyor.yml

```diff
@@ -17,10 +17,11 @@ environment:
   SFDX_AUTOUPDATE_DISABLE: true
   SFDX_DOMAIN_RETRY: 300
   matrix:
-    - vscode: test:without-system-tests
-    #- vscode: test:insiders-without-system-tests
+    - vscode: test:unit-and-system-tests
+    - vscode: test:integration-tests-without-system-tests
```

Comment:
Job1 = unit tests and system tests; Job2 = integration tests and vscode integration tests.